### PR TITLE
Implemented GetActionsForFunctionDefinition #127

### PR DESCRIPTION
**Pull request by manick02**
Signed-off-by: manick02 <manickavasagam.sundaram@gmail.com>

**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:
Implemented GetActionForFunctionDefinition utility Method
**Special notes for reviewers**:

**Additional information (if needed):**

### CODE REVIEW

File: utils/src/main/java/io/serverlessworkflow/utils/WorkflowUtils.java [hunk 1]
```diff
@@ -258,6 +258,97 @@ public static FunctionDefinition getFunctionDefinitionsForAction(
...
+            }
+          }
+        }
+      }
+    }
+
+    return actions;
+  }
+
+  private static boolean doesActionUsesFuntionDefinition(
```
Comment (tsurdilo) [inline utils/src/main/java/io/serverlessworkflow/utils/WorkflowUtils.java]: could rename to actionUsesFunctionDefinition

**Review by tsurdilo — APPROVED** [review]

**Comment by tsurdilo** [comment]
@manick02 very nice! thank you again. Just left a small comment .

File: utils/src/main/java/io/serverlessworkflow/utils/WorkflowUtils.java (hunk 1 above)
Comment (cb-manick) [reply utils/src/main/java/io/serverlessworkflow/utils/WorkflowUtils.java]: I think i will rename to checkIfActionUsesFunctionDefinition, is this name fine?

Comment (tsurdilo) [reply utils/src/main/java/io/serverlessworkflow/utils/WorkflowUtils.java]: sure :) sounds good

**Pull request merged**